server/api/teachables.js

```diff
@@ -0,0 +1,9 @@
+const router = require('express').Router()
+const { Teachables } = require('../db/models');
+module.exports = router
+
+router.get('/', (req, res, next) => {
+  Teachables.findAll({ include: [{ all: true }] })
+    .then(teachables => res.status(200).json(teachables))
```

Comment:
A global observation re your routes: 200 is the default response status, so there's no need to explicitly set it. You can just `res.json()`.

Reply:
Changed!✔️